docs/Analyzer Configuration.md

```diff
@@ -392,3 +392,20 @@ Option Values: integral values
 Default Value: Specific to each configurable rule ('100000' by default for most rules)
 
 Example: `dotnet_code_quality.CA5387.sufficient_IterationCount_for_weak_KDF_algorithm = 100000`
+
+#### Do not prefix enum values with type name
+Option Name: `enum_values_prefix_trigger`
+
+Configurable Rules: [CA1712](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1712)
+
+Option Values:
+
+| Option Value | Summary |
+| --- | --- |
+| `AnyEnumValue` | The rule will be triggered if _any_ of the enum values starts with the enum type name. |
```

Comment:
Also feel free to suggest better options and/or description.

Reply:
Looks good to me.